Bindings/Python/CMakeLists.txt

```diff
@@ -6,6 +6,12 @@
 find_package(PythonInterp 2.7 REQUIRED)
 find_package(PythonLibs 2.7 REQUIRED)
 
+# We may need to update the python install dir to include the python version,
```

Comment:
This maybe unrelated but seems like python 3 is widely supported now, would the above work for python 3?